Project 2 (13.12)/script.js

```diff
@@ -0,0 +1,98 @@
+//          1st task
+const reverseString = (text) => {
+  const newText = text.split("").reverse().join("");
+  return newText;
+};
+
+reverseString("Agnieszka");
+
+//            2nd task
+const changeFirstCharacter = (string) => {
+  const firstLetterFirstWord = string.split(" ")[0].slice(0, 1);
+  const firstLetterSecondWord = string.split(" ")[1].slice(0, 1);
+
+  const newFirstWord = firstLetterSecondWord + string.split(" ")[0].slice(1);
+  const newSecondWord = firstLetterFirstWord + string.split(" ")[1].slice(1);
+
+  const result = newFirstWord + " " + newSecondWord;
+  return result;
+};
+
+changeFirstCharacter("Marek Jurek");
+
+//          3rd task
+const volumeDifference = () => {
+  const firstArr = [1, 2, 3];
+  const secondArr = [4, 2, 8];
```

Comment:
to powinno być bardziej jako argumenty funkcji, bo w takim układzie Twoja funkcja jest trochę jednorazowego użytku :)